tools/simple_test.sh

```diff
@@ -0,0 +1,68 @@
+#!/bin/bash
+# ------------------------------------------------------------------------------
+# This is meant to be a somewhat self contained script for quickly
+# proving an Ethereum mainnet block with the type 1 prover. The goal
+# is to use this for benchmarking and CI. This is the block in
+# question: https://etherscan.io/block/19240704
```

Comment:
Given the speed of execution, aren't we testing `https://etherscan.io/block/19240705` instead? i.e. the block with 13 txns?

Reply:
Yeah nice catch. I've addressed in a new PR:
https://github.com/0xPolygonZero/zero-bin/pull/65

This one is a little weird because it was initially based on `testing`